### PR TITLE
Add DR test profile on vSphere 8.x for 4.16

### PR DESCRIPTION
**Pull request by wangke19**
Add DR test profile on vSphere 8.x for OCP 4.16. Associated Jira issue https://issues.redhat.com/browse/OCPQE-20102

### CODE REVIEW

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-8-upi-private-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-8-upi-private-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-8-upi-private-proxy-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-8-ipi-private-proxy-f14-disasterrecovery

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-vsphere-8-ipi-private-proxy-f14-disasterrecovery

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml [hunk 1]
```diff
@@ -640,6 +640,17 @@ tests:
     - ref: idp-htpasswd
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi-private
+- as: vsphere-8-ipi-private-proxy-f14-disasterrecovery
+  cron: 11 11 22 * *
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml:644]: The test name is `f14`, while the cron entry indicates its f28. Perhaps make them in sync ?

Comment (wangke19) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml:644]: Yes, notice this wrong setting.

Comment (wangke19) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml]: Updated.

**Pull request merged**